asset/css/doc.css

```
@@ -273,30 +285,43 @@ span.icon-expand:not(.open) > .navmap-tag , span.no-expand > .navmap-tag {
   border-color: rgb(32, 68, 165);
 }
 
-span.no-expand > .navmap-tag {
-  margin-left: 12px;
+span.icon-expand>.navmap-tag,
+span.no-expand>.navmap-tag {
```

```suggestion
span.icon-expand > .navmap-tag,
span.no-expand > .navmap-tag {
```

for better readability